src/lib/containers/entity_finder/EntityFinder.tsx

```diff
@@ -138,7 +151,7 @@ export const EntityFinder: React.FunctionComponent<EntityFinderProps> = ({
   showTypes,
   selectableTypes = Object.values(EntityType),
   selectedCopy = 'Selected',
-  visibleTypesInTree = [EntityType.PROJECT, EntityType.FOLDER],
+  visibleTypesInTree = DEFAULT_VISIBLE_TYPES,
```

Comment:
For some reason, this array literal gets reinitialized over-and-over, causing the component to repeatedly re-render! Switching to a constant fixes the issue.